### DISCUSSION TITLE
How to filter a list and highlight the active term

### DISCUSSION THREAD

**Discussion by hansipete** [unpoly/unpoly#553] [Q&A]
Hello there, I really enjoy using Unpoly. Thank you for that amazing piece of software!

There's one thing that I can't figure out right now. I have that section on my page where I filter a collection of items by term. Pseudocode looks like this:

```
<div class="terms">
 <a href="items" up-target=".results">All</a>
 <a href="items/tag:bananas" up-target=".results">Bananas</a>
 <a href="items/tag:apples" up-target=".results">Apples</a>
</div>
<div class="results">
 ... 
</div>
```

Everything works like expected but I can't figure out how to indicate the active term. Tried with up-nav tags but can't see the .up-current class attached...

Thank you for you help,
Hans

**Comment by adam12** [comment]
> Tried with up-nav tags but can't see the .up-current class attached

Perhaps it's the `:` causing the issue. What happens if you did an `up-alias` with the same value?

ie.

```
<div class="terms" up-nav>
   <a href="items/tag:bananas" up-target=".results" up-alias="items/tag:bananas">Bananas</a>
</div>
```

**Comment by triskweline** [comment]
Note that `.up-current` classes are only assigned if the link's `[href]` matches the current browser location in the address bar.

The links in your pseudo-code look like they do not update the browser location. [By default Unpoly only changes history when a main element is rendered](https://unpoly.com/updating-history#only-major-fragments-change-history). You can force a history-change with an `[up-history=true]` attribute.
